languages/python/oso/polar/host.py

```diff
@@ -204,5 +214,20 @@ def to_python(self, value):
             )
         elif tag == "Variable":
             return Variable(value[tag])
+        elif tag == "Expression":
```

Comment:
Is there a way we can make sure these don't get handled for regular values?

Reply:
What happens if you do:
```
❯ python -m oso
query> x = (1 and 2)
UnexpectedPolarTypeError
Expression
```

Reply:
Not exactly sure what you mean here, but `x = (1 and 2)` isn't a well-formed query. The fact that it gets an error in Python but not, apparently, in some of the other languages (Ruby, JS) is (I think) a non-partial-eval-related bug.

Reply:
Oh that's weird.

Yeah, the example is contrived, but just worried that by now permitting expressions to be converted to Python we might lose an error case.